public/index.php

```diff
@@ -9,17 +9,28 @@
 use Slim\Http\Response;
 use Slim\App;
 
-// > Env
-// Load "./.env" file as the "$_ENV" variable
-$dotenv = new Dotenv\Dotenv(realpath('..'));
-$dotenv->load();
-// ---
-
 // Create the Slim App
 $app = new App();
 
+// CORS
```

Comment:
Two sites such as the server and the prototypes or the server and the producer website